specs/ems/estimation-measurement-vocab-1-0.html

```diff
@@ -0,0 +1,2155 @@
+        origin point in time. For example, in the Gregorian calendar, points in time are measured as the amount of time
+        that has elapsed since a certain historical event that is denoted as year 1 AD. Points in <a
+          href="http://en.wikipedia.org/wiki/Unix_time">Unix time</a> as measured as the elapsed time since the
+        so-called <em>epoch</em> which has the Gregorian date 1970-01-01T00:00:00Z.</p>
+
+      <h6 id="Table-of-ems.3ATimeMetric-Standard-URIs">Table of ems:TimeMetric Standard URIs</h6>
+
+      <table>
+        <thead>
+          <tr>
+            <th><strong>URI</strong></th>
+            <th><strong>Description</strong></th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>metric:Duration</td>
+            <td>The amount of time that passes between two events, e.g. the start and finish of a project or task.</td>
+          </tr>
+          <tr>
+            <td>metric:Start</td>
+            <td>The point in time that marks the start of some activity or process, e.g. the start of a project or task.
+            </td>
+          </tr>
+          <tr>
+            <td>metric:Finish</td>
+            <td>The point in time that marks the finish of some activity or process, e.g. the finish of a project or
+              task.</td>
+          </tr>
+        </tbody>
+      </table>
+
+      <section id="metric.3ADuration">
+
+        <h4 id="metric.3ADuration">metric:Duration</h4>
+
+        <p>metric:Duration measures the amount of time between the start and finish of a project. It is typically broken
+          down by task and phase dimensions.</p>
+
+      </section>
+
+      <section id="metric.3AStart">
+
+        <h4 id="metric.3AStart">metric:Start</h4>
+
+        <p>metric:Start measures the point in time at which the project starts. It is typically broken down by task and
+          phase dimensions.</p>
+
+      </section>
+
+      <section id="metric:Finish">
+
+        <h4 id="metric.3AFinish">metric:Finish</h4>
+
+        <p>metric:Finish measures the point in time at which the project finishes. It is typically broken down by task
+          and phase dimensions.</p>
+
+      </section>
+
+      <section id="Time-Units-of-Measure">
+
+        <h4 id="Time-Units-of-Measure">Time Units of Measure</h4>
+
+        <p>The metric metric:Duration is the kind of quantity identified by <a
+            href="http://www.qudt.org/qudt/owl/1.0.0/quantity/Instances.html#Time">qudt-quantity:Time</a>. QUDT defines
+          several units of measure for it. We introduce additional standard URIs for a more complete list of time units,
+          but define them in terms of QUDT.</p>
+
+        <p>QUDT does not describe the concept of a point in time such as Gregorian date-time or UNIX time.</p>
+
+        <p>The following table lists the QUDT units for measuring time durations:</p>
```

Comment:
why not just make a normative reference to QUDT? Who are we to republish their copyrighted content?

cc @jamsden @ndjc